opencog/guile/SchemePrimitive.h

```diff
@@ -93,23 +93,34 @@ class SchemePrimitive : public PrimitiveEnviron
 			bool (T::*b_b)(bool);
 			bool (T::*b_hi)(Handle, int);
 			bool (T::*b_hh)(Handle, Handle);
+                        bool (T::*b_sddd)(const std::string&,double,double,double);
+                        bool (T::*b_sdii)(const std::string&,double,int,int);
+                        bool (T::*b_shddd)(const std::string&,Handle,double,double,double);
+                        bool (T::*b_siddd)(const std::string&,int,double,double,double);
```

Comment:
@yantrabuddhi could fix the indentation, they should be tabs not whitespaces. (Curious, what text editor / IDE do you use?)